tests/unit_tests/udf_framework/test_json_udf_query_handler.py

```diff
@@ -1,43 +1,22 @@
 import json
+import pytest
+
 from json import JSONDecodeError
-from pathlib import PurePosixPath
 from typing import Union
 
-import pytest
-from exasol_bucketfs_utils_python.localfs_mock_bucketfs_location import LocalFSMockBucketFSLocation
 from exasol_data_science_utils_python.schema.column import Column
 from exasol_data_science_utils_python.schema.column_name import ColumnName
 from exasol_data_science_utils_python.schema.column_type import ColumnType
 
 from exasol_advanced_analytics_framework.query_handler.context.scope_query_handler_context import \
     ScopeQueryHandlerContext
-from exasol_advanced_analytics_framework.query_handler.context.top_level_query_handler_context import \
-    TopLevelQueryHandlerContext
 from exasol_advanced_analytics_framework.query_handler.json_udf_query_handler import JSONQueryHandler, JSONType
 from exasol_advanced_analytics_framework.query_handler.result import Continue, Finish
 from exasol_advanced_analytics_framework.query_result.mock_query_result import MockQueryResult
 from exasol_advanced_analytics_framework.query_result.query_result import QueryResult
 from exasol_advanced_analytics_framework.udf_framework.json_udf_query_handler_factory import JsonUDFQueryHandler
 
 
-@pytest.fixture()
-def temporary_schema_name():
-    return "temp_schema_name"
-
-
-@pytest.fixture()
-def top_level_query_handler_context(tmp_path,
```

Comment:
Removing local fixtures may lead to weird behavior depending on which global fixture is used. Make sure that the global fixture fits and that change to it, doesn't break this test, because test could make specific assumptions

Reply:
And, if we use a global it should be more precise in what it provides. I think, I started with that at some point already.

Reply:
See the following link as an example 

https://github.com/exasol/machine-learning-library/blob/logistic_regression/tests/fixtures/top_level_query_handler_context_fixture.py

However, I suggest to introduce this in a separate PR

Reply:
This will be addressed in ticket  #188.